extensions/database/src/com/google/refine/extension/database/DatabaseConfiguration.java

```diff
@@ -42,8 +42,8 @@
     
     //optional parameters
     private boolean useSSL;
-    
-    
+
+
```

Comment:
Could you perhaps cancel this spurious and unrelated change?

Reply:
Sure, sorry for that.